schema.proto

```diff
@@ -40,6 +46,20 @@ message Block {
   repeated FactV2 facts_v2 = 4;
   repeated RuleV2 rules_v2 = 5;
   repeated CheckV2 checks_v2 = 6;
+  repeated Scope scope = 7;
+  repeated PublicKey publicKeys = 8;
+}
+
+message Scope {
+  enum ScopeType {
+    Authority = 0;
+    Previous  = 1;
+  }
+
+  oneof {
```

Comment:
looks like protoc is complaining about the absence of name for that `oneof`. What was it supposed to be named?

Reply:
We used `Content` in similar cases (`Proof`, `TermV2`, `Op`)